integration/apps/hpl_cpu/hpl_cpu.py

```diff
@@ -116,8 +127,23 @@ def get_bash_setup_commands(self):
         EOF
         '''.format(N=self.N, NBs=self.NBs, P=self.P, Q=self.Q))
 
-        setup_commands = 'export MKL_NUM_THREADS={cpu_per_rank}\n'.format(cpu_per_rank=self._cpu_per_rank)
-        setup_commands += 'cat > ./HPL.dat << EOF {input_file}\n'.format(input_file=input_file)
+        setup_commands = 'cat > ./HPL.dat << EOF {input_file}\n'.format(input_file=input_file)
+
+        # Below are only needed if the codebase is MKL.
+        if not self.mklroot is None:
```

Comment:
```if self.mklroot is not None```

Reply:
This could just as easily be ```if codebase == "mkl"```.  Then you don'e need ```self.mklroot``` at all.

Reply:
Done. That was a left over from a previous version, thanks for catching it.